tools/gsutil/BUILD

```
@@ -0,0 +1,24 @@
+load("//bazel:envoy_build_system.bzl", "envoy_package")
+load("@base_pip3//:requirements.bzl", "requirement")
+load("@rules_python//python:defs.bzl", "py_binary", "py_library")
+
+licenses(["notice"])  # Apache 2
+
+envoy_package()
+
+# Please see ./tools/gsutil/vendor_util.sh for a Docker-based
+# method of updating this vendored lib.
+py_library(
+    name = "crcmod",
+    srcs = glob(["crmod/*.py"]),
```

```suggestion
    srcs = glob(["crcmod/*.py"]),
```

interesting that this didnt fail - perhaps only the `.so` is required - ill test